tests/turbo.node.test.ts

```diff
@@ -200,6 +200,16 @@ describe('Node environment', () => {
         const response = await turbo.uploadFile({
           fileStreamFactory: () => fs.createReadStream(filePath),
           fileSizeFactory: () => fileSize,
+          opts: {
+            target: '43charactersAbcdEfghIjklMnopQrstUvwxYz12345',
```

Comment:
Since we're doing base64url conversion, try with invalid characters, with/without underscores and minus signs, etc.

Reply:
7ee5a8f